Code/Framework/AzToolsFramework/AzToolsFramework/Application/ToolsApplication.cpp

```diff
@@ -396,6 +396,8 @@ namespace AzToolsFramework
                 ->Event("IsSelected", &ToolsApplicationRequests::IsSelected)
                 ->Event("AreAnyEntitiesSelected", &ToolsApplicationRequests::AreAnyEntitiesSelected)
                 ->Event("GetSelectedEntitiesCount", &ToolsApplicationRequests::GetSelectedEntitiesCount)
+                ->Event("CreatePrefab", &ToolsApplicationRequests::CreatePrefab)
```

Comment:
ToolsApplication bus and the class already is doing a lot. To keep all the behavior context exposed calls in one place, I suggest creating a new bus instead of reusing ToolsApplication. Besides, it would be a good idea to not let ToolsApplication be filled with a lot of prefab-specific code. Would be difficult to untangle it later for future changes to the system.

Reply:
Fixed.